cmd/node/config/economics.toml

```diff
@@ -16,7 +16,7 @@
          {Year = 11, MaximumInflation = 0.0},
     ]
     Denomination = 18 # represents the smallest eGLD subdivision (10^-X eGLD for a denomination of X)
-    GenesisMintingSenderAddress = "erd1lllllllllllllllllllllllllllllllllllllllllllllllllllsckry7t"
+    GenesisMintingSenderAddress = "erd17rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rcq4288pn"
```

Comment:
please change this to `erd17rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rc0pu8s7rcqqkhty3` as the original address is malformed and the decode will raise an `invalid checksum (expected qkhty3 got 4288pn)` error

Reply:
Ok. Changed the address